tests/test_header_items.py

```diff
@@ -0,0 +1,73 @@
+import os, sys; sys.path.insert(0, os.path.dirname(os.path.dirname(__file__)))
+
+from pprint import pformat
+
+import pytest
+
+import lasio
+import lasio.las_items
+
+
+test_dir = os.path.dirname(__file__)
+
+egfn = lambda fn: os.path.join(os.path.dirname(__file__), "examples", fn)
+
+stegfn = lambda vers, fn: os.path.join(
+    os.path.dirname(__file__), "examples", vers, fn
+)
+
+
+def test_repr():
+    h = lasio.las_items.HeaderItem(
+        'MN', unit='m', value=20, descr='test testing')
+
+    assert h.__repr__() == pformat(h)
+
+
+def test_useful_mnemonic_setter_not_allowed():
+    h = lasio.las_items.HeaderItem(
+        'MN', unit='m', value=20, descr='test testing')
+
+    with pytest.raises(ValueError):
+        h.useful_mnemonic = "NEW_NAME"
+
+
+def test_mmenomic_names_behavior():
```

Comment:
Is this the expected behavior?

Reply:
Yep useful_mnemonic is only there to handle duplicate or empty mnemonics.